### PR TITLE
Loader

### PR DESCRIPTION
**Pull request by scraperdragon**
xypath: improved loading of messytables from file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 *.py[cod]
 fixtures/.*
-
+src
```
Comment (frabcus) [inline .gitignore]: What's src/ for, out of interest?

File: test/test_loader.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import xypath.loader
```
Comment (frabcus) [inline test/test_loader.py]: None of these tests run for me with `nosetests`. 

Think you should wrap them in a test class so they do!

File: .gitignore (hunk 1 above)
Comment (scraperdragon) [inline .gitignore]: The way I've got messytables installed at the moment is that it's in a
src/messytables directory off xypath. So I want to ignore its existence.

File: test/test_loader.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+from nose.tools import assert_equal
+fn = "fixtures/4132.zip"
+
+tables = xypath.loader.table_set(fn)
+
+four, = xypath.loader.get_sheets(tables, 0)
```
Comment (frabcus) [inline test/test_loader.py]: Am assuming the order in the zip file is deliberate:

```
bat:fixtures francis$ unzip -l 4132.zip 
Archive:  4132.zip
  Length     Date   Time    Name
 --------    ----   ----    ----
        5  01-16-15 11:36   four.csv
        4  01-16-15 11:36   one.csv
        6  01-16-15 11:36   three.csv
        4  01-16-15 11:36   two.csv
 --------                   -------
       19                   4 files
```

Might be worth noting in a comment

File: xypath/loader.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import messytables
+import xypath
+
+def table_set(filename):
```
Comment (frabcus) [inline xypath/loader.py]: I'm a bit confused by the interface here, in particular the relationship between `table_set` and `get_sheets`.

I'd expect table_set to return all the tables (table == sheet in my mental model!), I don't understand why there's a second function that can then return multiple things.

Comment (scraperdragon) [inline xypath/loader.py]: `table_set` does indeed return all the tables.
But then how do you get a subset of those tables? That's `get_sheets` job. Writing docstrings.

Comment (scraperdragon) [inline xypath/loader.py]: Done.

File: test/test_loader.py (hunk 2 above)
Comment (scraperdragon) [inline test/test_loader.py]: Done.

**Pull request merged**